### DISCUSSION TITLE
Shortcode

### DISCUSSION THREAD

**Discussion by dougie-wougie** [code4recovery/12-step-meeting-list#1154] [Q&A] [answered]
I have a template page (Divi framework) and want to use the shortcut to show what I see on the /meetings page. I get the known to me shortcuts to work, but I do not want a count anything or limit the listing, such as count=5. Is there a shortcut that acts like the meetings page?

Dougie

**Comment by joshreisner** [comment, marked as answer]
hey dougie, yes there is a shortcode called `[tsml_ui]` that is similar to the meetings page

**Reply by jpiemeisl** [reply]
I noticed the shortcode `[tsml_ui]` is the list view but does not have the list/map toggle feature. Is there an option to enable the map view using the shortcode? Thank you!

**Reply by joshreisner** [reply]
are you using google maps? TSML UI only supports the mapbox option - if you get a mapbox access token the map view will start showing up in TSML UI

**Reply by dougie-wougie** [reply]
I am using Mapbox. I am not a big supporter of Google. Thanks for the help.

**Reply by jpiemeisl** [reply]
@joshreisner switching to Mapbox worked. Thank you so much!

**Comment by dougie-wougie** [comment]
Awesome. Thank you. A suggestion, could it be added to the FAQ section?

Josh, I know you will not remember me. We corresponded about seven years ago. When we even talked on the phone, you invited me to a recovery code gathering in San Jose. I was working on our Alano club's site, and 12-step-meeting-list was to be used at the Central Office level. Well, today, I am doing the CO site rework. I am back in Santa Clarita from Portland (I was there for over five years).

Doug

**Reply by joshreisner** [reply]
Hey Doug, good to hear from you again, welcome back to CA

Sure we can add `[tsml_ui]` to the FAQ item about shortcodes. I created an issue to track it here: https://github.com/code4recovery/12-step-meeting-list/issues/1155
